src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1947 @@
+
+class HSI_Wasting_SupplementaryFeedingProgramme_MAM(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is the supplementary feeding programme for MAM without complications
+    """
+
+    def __init__(self, module, person_id):
+        super().__init__(module, person_id=person_id)
+
+        # Get a blank footprint and then edit to define call on resources
+        # of this treatment event
+        the_appt_footprint = self.sim.modules["HealthSystem"].get_blank_appt_footprint()
+        the_appt_footprint['Under5OPD'] = 1  # This requires one outpatient
+
+        # Define the necessary information for an HSI
+        self.TREATMENT_ID = 'Undernutrition_Feeding_Supplementary'
+        self.EXPECTED_APPT_FOOTPRINT = the_appt_footprint
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+        self.ALERT_OTHER_DISEASES = []
+
+    def apply(self, person_id, squeeze_factor):
+        assert isinstance(self.module, Wasting)
+
         df = self.sim.population.props
-        df.at[child, 'un_clinical_acute_malnutrition'] = 'well'
-        df.at[child, 'un_ever_wasted'] = False
+
+        # no treatment if already dead
+        if not df.at[person_id, 'is_alive']:
+            return
+
+        # Do here whatever happens to an individual during the admission for the treatment
+        # ~~~~~~~~~~~~~~~~~~~~~~
+        # Perform measurements (height/length), weight, MUAC
+        self.add_equipment({'Height Pole (Stadiometer)', 'Weighing scale', 'MUAC tape'})
+
+        # Check and log availability of consumables
+        if self.get_consumables(item_codes=self.module.cons_codes['SFP']):
+            logger.debug(key='debug', data='consumables are available')
+            # Record that the treatment is provided:
+            df.at[person_id, 'un_am_treatment_type'] = 'CSB++'
+            self.module.do_when_am_treatment(person_id, intervention='SFP')
+        else:
+            logger.debug(key='debug',
+                         data=f"Consumable(s) not available, hence {self.TREATMENT_ID} cannot be provided.")
+
+    def did_not_run(self):
+        logger.debug(key='debug', data=f'{self.TREATMENT_ID}: did not run')
+        pass
+
+
+class HSI_Wasting_OutpatientTherapeuticProgramme_SAM(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is the outpatient management of SAM without any medical complications
+    """
+
+    def __init__(self, module, person_id):
+        super().__init__(module, person_id=person_id)
+
+        # Get a blank footprint and then edit to define call on resources
+        # of this treatment event
+        the_appt_footprint = self.sim.modules["HealthSystem"].get_blank_appt_footprint()
+        the_appt_footprint['U5Malnutr'] = 1
+
+        # Define the necessary information for an HSI
+        self.TREATMENT_ID = 'Undernutrition_Feeding_Outpatient'
+        self.EXPECTED_APPT_FOOTPRINT = the_appt_footprint
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+        self.ALERT_OTHER_DISEASES = []
```

Comment:
```suggestion
```
This has been deprecated for a long time, please remove the 'ALERT_OTHER_DISEASES' declarations.